### ISSUE TITLE
Feedback on import helpers feature

### ISSUE THREAD

**Issue by virgile-dev** [umap-project/umap#1926]
First of all, congrats ! It's amazing to create maps in 2 minutes that I used to create in like half an hour. I'm really excited to see how it evolves. Here is my first creation  : https://dev.umap-project.org/en/map/chauffailles-services-publics-et-defibrillateurs_6506

Feedback bellow:
I find the icons and associated text not super clear. I tested each of them to understand what they did.
For example : 
- "Contours communes" might be clearer than "Communes". Maybe using a real contour of a commune for the icon would indicate a bit more the idea ?
- GeoDataMine is not really well known project. Maybe using "OSM Thematic Data" would make it more understandable. And using the OSM Icon would indicate the provenance of the data
- I didn't expect Regions and Departments contours behind the Datasets icon. I'm wondering if it wouldn't be better to have a single button called "Contours" or "Administrative contours" where the user can select a Communes, all communes, a department, all departement, a region, all regions. But I'm guessing there is an idea to add popular datasets in the dropdown. Maybe adding a helper sending the user someplace to ask for a dataset to be added ?
- Maybe a name like Overpass (OSM Query) would help everyone understand. Overpass while being more well know could benefit from a small explanation of what it is inside the modal ("A way to query OSM data".  I didn't read the full tutorial but it seemed a bit complicated to learn, maybe on explore we could provide a few examples and redirect to the full tutorial. For example I was able to know what to type for defibrillators thanks to this tutorial. So maybe redirecting to a dictionnary of OSM terminology might me more useful to users ?

It's only after I made my first query through geodatamine and overpass that I understood what it meant "Simplify all geometries to points." because my request displayed areas. 

Lastly, does this feature is hidden behind a feature flag that the admin activates for its instance ?

**Comment by yohanboniface** [comment]
> Lastly, does this feature is hidden behind a feature flag that the admin activates for its instance ?

Yes it is. For eg. in OSM fr, only Overpass is activated for now.

**Comment by Aurelie-Jallut** [comment]
The headings under the buttons have changed, so I suggest we adapt the pictograms too. 
What do you think of these? 
![image](https://github.com/user-attachments/assets/3140b0ea-2c25-46b2-8a2e-5b2811348c5b)

**Comment by Aurelie-Jallut** [comment]
![departements-regions](https://github.com/user-attachments/assets/c7bc4234-302c-45b2-b7ba-7db01e067a8d)
![communes-france](https://github.com/user-attachments/assets/21765346-f339-44d4-ad48-a7d69e2c46c8)

**Comment by Aurelie-Jallut** [comment]
we could add a help picto after the title:
1- Choose the data (?)
or after Assistant d'import

and in the help popup write :
uMap allows you either to import data freely (a file, a URL or by copy/paste) or to use the import wizard which will automatically fill in the "Fill in a URL" field.

in french : 
uMap permet soit d’importer librement des données (un fichier, une URL ou par copier/coller), soit d’utiliser l’assistant d’import qui remplira automatique le champ “_Renseigner une URL_”.

![image](https://github.com/user-attachments/assets/211a5a57-9cd6-41f5-b09f-9b9fdc7d6ed3)

**Comment by davidbgk** [comment]
It's easy to change the Communes pictogram but it's less trivial to change the dataset (Régions, départements France) one because it's intended to be generic. Or maybe we create a new dedicated entry for that one and keep dataset generic (but unused at the moment)? 🤔
